regular-expressions/introduction/basic-syntax-i/regex-wildcards-ii.md

```diff
@@ -0,0 +1,84 @@
+---
+author: emmab
+
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+---
+
+# Wildcards II
+
+---
+## Content
+
+Going back to our previous example - if we wanted to find all names beginning with `Fre` then we could write our regex pattern as:
+
+`/Fre\w*/`
+
+This would ensure that the names start with `Fre`, and then have zero or more word characters. 
```

Comment:
should we say match instead of ensure?
```suggestion
This would match the names that start with `Fre`, and then have zero or more word characters. 
```